README.md

```diff
@@ -12,11 +12,14 @@ Requires:
 ### Configuration
 Configuration values can be set either from environmental variables or code:
 
-`SIGNALFX_SERVICE_NAME` / [WithServiceName](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithServiceName) Name identifying the service as a whole (defaults to `SignalFx-Tracing`)
-
-`SIGNALFX_ENDPOINT_URL` / [WithEndpointURL](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithEndpointURL) URL to send traces to (defaults to `http://localhost:9080/v1/trace`)
-
-`SIGNALFX_ACCESS_TOKEN` / [WithAccessToken](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithAccessToken) (no default)
+| Code | Environment Variable | Default Value | Notes |
+| ---  | ---                  | ---           | ---   |
+| [WithServiceName](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithServiceName) | `SIGNALFX_SERVICE_NAME` | `SignalFx-Tracing` | Name identifying the service as a whole |
+| [WithEndpointURL](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithEndpointURL) | `SIGNALFX_ENDPOINT_URL` | `http://localhost:9080/v1/trace` | URL to send traces to  |
+| [WithAccessToken](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithAccessToken) | `SIGNALFX_ACCESS_TOKEN` | none | |
+| [WithGlobalTag](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithGlobalTag) | `SIGNALFX_SPAN_TAGS` | none | If specifying as an environment variable, the format is `key:value` with multiple values separated by commas. Ex.`"a:b, c:d"`|
+
+Note: The current transport format is Zipkin which only supports string values for tags.
```

Comment:
If this is for `SIGNALFX_SPAN_TAGS`, let's move it to the note for that config value.